static/js/user_profile.js

```diff
@@ -192,6 +214,62 @@ export function register_click_handlers() {
         e.preventDefault();
     });
 
+    $("body").on("submit", "#user-profile-modal .subscription_list_remove form", (e) => {
+        e.preventDefault();
+        const stream_row = $(e.currentTarget).closest("tr");
```

Comment:
I'd write this as looking for the closest `[data-stream-id]`, since that'll be more robust to HTML refactoring in the future.